README.md

```diff
@@ -8,6 +8,14 @@ Pony dependency management command-line tool.
 
 Corral is alpha level software. We advise you use [stable](https://github.com/ponylang/pony-stable) for your dependency management needs. We are looking for help building out Corral and would love if you help out with any of the [open issues](https://github.com/ponylang/corral/issues).
 
+## Instalation
+
+The following command will download the latest release of `corral` and install it to `~/.pony/ponyup/bin` by default.
```

Comment:
This should probably include a link to the ponyup repo, in case they don't already have that installed.

Reply:
Improved a bit, let me know what you think.